packages/e2e-tests/specs/block-hierarchy-navigation.test.js

```diff
@@ -96,8 +98,11 @@ describe( 'Navigating the block hierarchy', () => {
 		await openBlockNavigator();
 		await page.keyboard.press( 'Space' );
 
-		// Replace its content.
-		await pressKeyWithModifier( 'primary', 'A' );
+		// Replace its content
+		// note Cmd/Ctrl + A doesn't work on Mac with Pupetter right now
+		// https://github.com/GoogleChrome/puppeteer/issues/1313
+		await pressKeyTimes( 'ArrowRight', textString.length );
+		await pressKeyTimes( 'Backspace', textString.length );
```

Comment:
You can also do something like:
```
await pressKeyWithModifier( 'shift', 'End' );
await editorPage.keyboard.press( 'Backspace' );
```

Reply:
I'm afraid that doesn't work for me. I believe it's an issue with Pupeteer (see commit which fixed this) but it won't pass with anything other than explicit undo.